kit/src/main/java/com/oracle/javafx/scenebuilder/kit/editor/panel/library/maven/MavenDialogController.java

```diff
@@ -109,6 +109,7 @@ public class MavenDialogController extends AbstractFxmlWindowController {
     };
 
     private final PreferencesControllerBase preferencesControllerBase;
+    private boolean confirmed;
```

Comment:
Please update line 2 from `* Copyright (c) 2016, 2017 Gluon and/or its affiliates.` into `* Copyright (c) 2016, 2022 Gluon and/or its affiliates.`.